lib/notesLinks.ml

```diff
@@ -59,66 +59,68 @@ let misc_notes_link s i =
-      let (spe, b) =
-        try
-          let i = String.index b ':' in
-          Some (String.sub b 0 i),
-          String.sub b (i + 1) (String.length b - i - 1)
-        with Not_found -> None, b
-      in
-      let (b, text) =
-        try
-          let i = String.rindex b ';' in
-          String.sub b 0 i,
-          Some (String.sub b (i + 1) (String.length b - i - 1))
-        with Not_found -> b, None
-      in
-      if spe = Some "w" then
-        let (wiz, name) =
-          match try Some (String.index b '/') with Not_found -> None with
-            Some i ->
-              String.sub b 0 i, String.sub b (i + 1) (String.length b - i - 1)
-          | None -> b, ""
+      if j > i + 4 then
+        let b = String.sub s (i + 2) (j - i - 4) in
+        let (spe, b) =
+          try
+            let i = String.index b ':' in
+            Some (String.sub b 0 i),
+            String.sub b (i + 1) (String.length b - i - 1)
+          with Not_found -> None, b
         in
-        WLwizard (j, wiz, name)
-      else
-        try
-          let k = 0 in
-          let l = String.index_from b k '/' in
-          let fn = String.sub b k (l - k) in
-          let k = l + 1 in
-          let (fn, sn, oc, name) =
-            try
-              let l = String.index_from b k '/' in
-              let sn = String.sub b k (l - k) in
-              let (oc, name) =
-                try
-                  let k = l + 1 in
-                  let l = String.index_from b k '/' in
-                  let x = String.sub b k (l - k) in
-                  x, String.sub b (l + 1) (String.length b - l - 1)
-                with Not_found ->
-                  "", String.sub b (l + 1) (String.length b - l - 1)
-              in
-              let oc1 = try int_of_string name with Failure _ -> -1 in
-              let oc = try int_of_string oc with Failure _ -> 0 in
-              if oc1 = -1 then (fn, sn, oc, name)
-              (* else if not Wiki.wi_person_exists (fn, sn, oc1) then (fn, sn, oc, fn ^ " " ^ sn) *)
-              else (fn, sn, oc1, fn ^ " " ^ sn)
-            with Not_found ->
-              let sn = String.sub b k (String.length b - k) in
-              let name = fn ^ " " ^ sn in fn, sn, 0, name
+        let (b, text) =
+          try
+            let i = String.rindex b ';' in
+            String.sub b 0 i,
+            Some (String.sub b (i + 1) (String.length b - i - 1))
+          with Not_found -> b, None
+        in
+        if spe = Some "w" then
+          let (wiz, name) =
+            match try Some (String.index b '/') with Not_found -> None with
+              Some i ->
+                String.sub b 0 i, String.sub b (i + 1) (String.length b - i - 1)
+            | None -> b, ""
           in
-          let fn = Name.lower fn in
-          let sn = Name.lower sn in WLperson (j, (fn, sn, oc), name, text)
-        with Not_found -> WLnone
+          WLwizard (j, wiz, name)
+        else
```

Comment:
This indentation change seems strange.

I think that the `      else WLnone` in line 123 be place before this one in order to close the opening `if j > i + 4 then`

Reply:
peut être c'est le `try ... with Not_found -> WLnone` qu'il faut supprimer?